Added Checklist and Known Issues section (#24)
dpc-sdp/sdp-docs: docs/troubleshooting.md

# Troubleshooting

!!! info
    
    You are viewing a version of the documentation that is currently under 
    active development.

>  This page provides some workaround for issues that you may encounter during a local build.

Reviewer: *workarounds

## Open Workspace | Cenitex | Citrix

> Unable to install software (e.g Docker Desktop) or run commands which requires elevated permissions.

Reviewer: *require

> 1. Check if you Open Workspace is in your machine. This will allow software installation that requires elevated permissions.

Reviewer: *Check if Open Workplace is installed on your machine.

Also, a screen shot of the PCInfo app would help here. Ping me if you need help with this.

> 2. If you believe to have Open Workspace installed but still have issue with installations, logoff from Citrix Gateway (Windows Taskbar Icon) and reconnect through Go Connect (Window search). An IE browser will open and ask for your DPC login credentials. Process might take some time to complete for others and for some, browser seemed to be hanging. In that case try restarting your computer and retry Go Connect again.

Reviewer: ```suggestion
2. If you believe Open Workplace is installed but are still having issues with installations, logoff from Citrix Gateway (Windows Taskbar Icon) and reconnect through Go Connect (Window search). An IE browser will open and ask for your DPC login credentials. Process might take some time to complete or your browser might appear to hang. In that case try restarting your computer and retry Go Connect.
```

> 3. Check again to see if you have the elevated permissions or not. You might have restart your computer several times to take effect.

Reviewer: *have to restart

Reviewer: *several times for it it to take effect.

> ## SSH Key

Reviewer: Delete if no issues.

*(add ssh key issue if any)*

## Lagoon | Lagoon CLI

> *(Remote work)* Unable to resolve host or cannot descrypt encoded SSH

Reviewer: *decrypt
